src/protocol/sort/reshare.rs

```diff
@@ -142,9 +142,11 @@ mod tests {
             let output_share = validate_and_reconstruct(f);
             assert_eq!(output_share, input);
 
-            assert_ne!(share[0], f.0);
-            assert_ne!(share[1], f.1);
-            assert_ne!(share[2], f.2);
+            if share[0] != f.0 && share[1] != f.1 && share[2] != f.2 {
+                new_reshares_atleast_once = true;
+                break;
+            }
         }
+        assert!(new_reshares_atleast_once);
```

Comment:
I mean, this isn't really what we want to test. We want to test that they ALL got re-shared. This is a stop-gap solution to fix the test failure, but I think longer-term you want to test with a bigger field.

Reply:
Yes lets fix it longer term to use a larger field instead